### PR TITLE
feat(installation.md): feat update to latest command

### PR DESCRIPTION
**Pull request by wiasliaw**
1. According to https://github.com/foundry-rs/foundry/pull/1037, current installation command will get error below.

```
error: multiple packages with binaries found: anvil, foundry-cli. When installing a git repository, cargo will always search the entire repo for any Cargo.toml. Please specify which to install.
```

2. provide update command to get the latest version of Foundry.

### CODE REVIEW

**Review by onbjerg — CHANGES_REQUESTED** [review]
Any way we can get it to install Anvil as well?

**Comment by gakonst** [comment]
@mattsse should we pull in the Anvil binary into `foundry-cli` so that all CLI bins are in the same package and avoid this issue?

**Comment by mattsse** [comment]
> @mattsse should we pull in the Anvil binary into `foundry-cli` so that all CLI bins are in the same package and avoid this issue?

if we do this then testing in `cli` will take get even longer.

I have no better solution yet, need to try something first.
the current setup of the `foundry-cli` is also not ideal, because it's essentially two crates in itself.

File: src/getting-started/installation.md [hunk 1]
```diff
@@ -47,9 +47,11 @@ Download and run `rustup-init` from [rustup.rs](https://rustup.rs). It will star
 After this, run the following to build Foundry from source:
 
 ```sh
-cargo install --git https://github.com/foundry-rs/foundry --bins --locked
+cargo install --git https://github.com/foundry-rs/foundry foundry-cli --bins --locked
```
Comment (onbjerg) [inline src/getting-started/installation.md:50]:
```suggestion
cargo install --git https://github.com/foundry-rs/foundry foundry-cli anvil --bins --locked
```

**Review by onbjerg — COMMENTED** [review]
To also install Anvil

**Pull request merged**